drop/modules/mae-pipeline/Snakefile

```diff
@@ -20,19 +20,22 @@ rule mae_dependency:
 rule sampleQC:
     input: cfg.getHtmlFromScript(MAE_WORKDIR / "QC" / "Datasets.R")
 
-rule create_dict_fa:
-    input:  "{fasta_file_base}.fa"
-    output: "{fasta_file_base}.dict"
-    params:
-        gatk = cfg.getTool("gatkCmd"),
-    shell: "{params.gatk} CreateSequenceDictionary --REFERENCE {input}"
+rule create_dict:
+    input:  cfg.genome.getFastaList()
+    output:  cfg.genome.getDictList()
+    shell: 
+	    """
+		for fasta in {input}
+		  do
+		  if [ -f "${{fasta%.*}}.dict" ]; then
+            echo "${{fasta%.*}}.dict exists."
+          else
+            echo "${{fasta%.*}}.dict does not exist."
+		    gatk CreateSequenceDictionary --REFERENCE $fasta
```

Comment:
why do we print this statement? if so, let's say "... .dict does not exist. We will create it"
but I would altogether remove printing